tests/integration/sync/test_sync_command.py

```diff
@@ -33,8 +33,9 @@
 SKIP_SYNC_TESTS = RUNNING_ON_CI and RUNNING_TEST_FOR_MASTER_ON_CI and not RUN_BY_CANARY
 IS_WINDOWS = platform.system().lower() == "windows"
 CFN_SLEEP = 3
+LAMBDA_SLEEP = 3
```

Comment:
Can we add some explanation why we need these sleeps? It is important to keep this recorded so that people can understand and adjust these values if it is needed